gateway-service-impl/src/main/java/org/hypertrace/gateway/service/common/util/MetricAggregationFunctionUtil.java

```diff
@@ -76,7 +76,8 @@ public static AttributeKind getValueTypeForFunctionType(
         // Min/Max/Sum function only applicable to numerical data
         Preconditions.checkArgument(
```

Comment:
Not sure if SUM makes sense for TIMESTAMP

Reply:
Okay